app/webpacker/components/Panel/SeniorDelegate/Regions/Subregion.jsx

```diff
@@ -37,6 +55,7 @@ export default function Subregion({ title, groupId }) {
   const [newDelegateUser, setNewDelegateUser] = useState(null);
   const [formError, setFormError] = useState(null);
   const { save, saving } = useSaveAction();
+  const confirm = useConfirm();
```

Comment:
Is there a `ConfirmProvider` available in the DOM tree of this component somewhere?

Reply:
Yes, in PanelTemplate. Since this is very widely used in different panels, I've already added it there.